content/Classes/Classes.q.md

```diff
@@ -0,0 +1,1088 @@
+
+ - `public function myMethod() { echo "This is a class method."; }`
+
+:::
+
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
+    	
+    $myObject = new MyClass;
+
+    echo $myObject->myProp;
+?>
+```
+/// type=SS, answer=[5]
+
+Execute the program. What is its output?
+
+ - It prints `MyClass`.
+
+ - It produces an error.
+
+ - No output is displayed.
+
+ - It prints `This is a class method.`.
+
+ - It prints `This is a class property.`.
+
+
+/// type=SS, answer=[1]
+
+On line 2, what does the `class` keyword do?
+
+ - It creates the `MyClass` class.
+
+ - It displays the `MyClass` class.
+
+ - It accesses the `MyClass` class.
+
+ - It defines the `MyClass` method.
+
+ - It creates the `MyClass` property.
+
+
+/// type=SS, answer=[4]
+
+Which of the following is a property?
+
+ - `class`
+
+ - `public`
+
+ - `MyClass`
+
+ - `$myProp`
+
+ - `myMethod()`
+
+
+/// type=SS, answer=[5]
+
+Which of the following is a method?
+
+ - `class`
+
+ - `public`
+
+ - `MyClass`
+
+ - `$myProp`
+
+ - `myMethod()`
+
+
+/// type=SS, answer=[2]
+
+In the statement `$myObject = new MyClass;` on line 12, what is `$myObject`?
+
+ - It is a method.
+
+ - It is an object.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=SS, answer=[4]
+
+On line 12, what does the statement `$myObject = new MyClass;` do?
+
+ - It sets the value of `$myObject` to `MyClass`.
+
+ - It replaces the value of `$myObject` with `MyClass`.
+
+ - It accesses the value of `MyClass` through `$myObject`.
+
+ - It creates the `$myObject` object of the `MyClass` class.
+
+ - It assigns the value of the `MyClass` class to `$myObject`.
+
+
+/// type=SS, answer=[5]
+
+On line 14, what does the statement `echo $myObject->myProp;` do?
+
+ - It creates the `myProp` property of `$myObject`.
+
+ - It sets the value of the `myProp` property of `$myObject`.
+
+ - It returns the value of the `myProp` property of `$myObject`.
+
+ - It removes the value of the `myProp` property of `$myObject`.
+
+ - It displays the value of the `myProp` property of `$myObject`.
+
+
+:::
+
+
+:::
+
+/// type=REPL
+
+```php
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
+    	
+    $myObject = new MyClass;
+
+    echo $myObject->myProp;
+?>
+```
+/// type=SS, answer=[4]
+
+On line 14, replace the statement `echo $myObject->myProp;` with `$myObject->myMethod();`. Execute the program. What is its output?
+
+ - It prints `MyClass`.
+
+ - It produces an error.
+
+ - No output is displayed.
+
+ - It prints `This is a class method.`.
+
+ - It prints `This is a class property.`.
+ 
+:::
+
+
+:::
+
+/// type=REPL
```

Comment:
Yes, this is only `/// type=REPL` since this is connected with the previous question when we allow the learners to edit the given code. This is to further engage the learners in the code and giving them the chance to make further changes. Hope this help. Thanks.